release-notes/7.0/7.0.0/api-diff/Microsoft.WindowsDesktop.App/7.0.0_System.Windows.Forms.Design.md

```diff
@@ -0,0 +1,54 @@
+# System.Windows.Forms.Design
+
+``` diff
+ namespace System.Windows.Forms.Design {
+     [ToolboxItemAttribute(false)]
+     public class ComponentEditorForm : Form {
+-        [BrowsableAttribute(false)]
+-        [EditorBrowsableAttribute(1)]
+-        public new event EventHandler AutoSizeChanged;
++        [BrowsableAttribute(false)]
++        [EditorBrowsableAttribute(1)]
++        public new event EventHandler? AutoSizeChanged;
+-        protected virtual void OnSelChangeSelector(object source, TreeViewEventArgs e);
++        protected virtual void OnSelChangeSelector(object? source, TreeViewEventArgs e);
+-        public virtual DialogResult ShowForm(IWin32Window owner);
++        public virtual DialogResult ShowForm(IWin32Window? owner);
+-        public virtual DialogResult ShowForm(IWin32Window owner, int page);
++        public virtual DialogResult ShowForm(IWin32Window? owner, int page);
+     }
+     public abstract class ComponentEditorPage : Panel {
+-        public Icon Icon { get; set; }
++        [AllowNullAttribute]
++        public Icon Icon { get; set; }
+     }
+     public interface IWindowsFormsEditorService {
+-        void DropDownControl(Control control);
++        void DropDownControl(Control? control);
+     }
+     public abstract class PropertyTab : IExtenderProvider {
+-        public virtual PropertyDescriptorCollection GetProperties(ITypeDescriptorContext? context, object component, Attribute[]? attributes);
```

Comment:
I do not see this change right. @carlossanlop can you double check line 30 becoming 31?  Here is what i see in 7.0 : https://github.com/dotnet/winforms/blob/35628d369b17a61dc9b564de30a7b6ae5d5285b1/src/System.Windows.Forms/src/System/Windows/Forms/Design/PropertyTab.cs#L123

Reply:
The only change I see is that the return value is nullable. I'll fix it in line 31.